spark/src/test/scala/org/apache/spark/sql/delta/DeltaColumnMappingSuite.scala

```diff
@@ -1907,8 +1895,9 @@ class DeltaColumnMappingSuite extends QueryTest
         s"""CREATE TABLE $testTableName
            |USING DELTA
            |TBLPROPERTIES(
-           |'$minReaderKey' = '2',
-           |'$minWriterKey' = '7'
+           |'$minReaderKey' = '3',
+           |'$minWriterKey' = '7',
+           |'${DeltaConfigs.ENABLE_DELETION_VECTORS_CREATION.key}' = 'true'
```

Comment:
You shouldn't need this change as part of this PR,  right? I initially made the minReaderKey 2 to expose a bug: https://github.com/delta-io/delta/commit/a655bed59e436dd0e9500a02441e65bfa9f103d6